### ISSUE TITLE
Accessibility: VoiceOver does not work well with adding files

### ISSUE THREAD

**Issue by joashrajin** [Automattic/pocket-casts-ios#2700]
### Description

> The issue I have found most recently is in files.
In particular in trying to share my own files.
Voiceover reads the “how do I do that” and “add file” buttons but if you double tap on either of those all you get are options for an existing file already in the list.

9313434-zd-a8c

### Step-by-step reproduction instructions

1. Enable VoiceOver
2. Go to Profile > Files > click the menu
3. Double tap on 'Add File'
4. Notice the issue

### Did you search for existing bug reports?

- [x] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

App Version: 7.80
Device: iPhone17,1
OS: 18.2.1
